omero/sysadmins/python3-migration.rst

```diff
@@ -0,0 +1,128 @@
+Migration to Python 3
+=====================
+
+.. warning::
+
+  This is still a pre-release of OMERO 5.6. The instructions below
+  are intended to help plan a migration path from your current OMERO
+  installation. We suggest you **not** perform the migration on your
+  production systems until the 5.6.0 release is ready.
+
+Basic steps
+-----------
+
+1. Choose a platform and a python. If your current installation platform
+   does not match one of the :doc:`recommended platforms <version-requirements>`,
+   you may want to choose a new platform as your migration target. See
+   `Choosing a platform`_ below.
+2. Install OMERO.server and OMERO.web *separately*. Though not necessary, all
+   instructions like `OMERO.server`_ and `OMERO.web`_ below as well as the main
+   :doc:`server <unix/server-installation>` and 
+   :doc:`web <unix/install-web/web-deployment>` installation pages now assume
+   that the two are in separate installations.
+3. Once both have been installed, perform a
+   :doc:`backup and restore <server-backup-and-restore>` procedure
+   and test your installation against the copy of your data.
+
+Choosing a platform
+-------------------
+
+We suggest Python 3.6. This is the default installation on the two recommended platforms,
+CentOS 7 and Ubuntu 18.04.
+
+Both 3.5 and 3.7 should work and are slated to have support added, but Python 3.6 has been
+the focus of testing during the migration.
+
+Similarly, other operating systems are slated for having support added, but help from the
+community would be very welcome! Obvious next candidates are CentOS 8 and Ubuntu 20.04.
+
+Debian is more of an issue. Debian 9 is still on Python 3.5 and Debian 10 has moved to Ice 3.7
+so neither is particularly well supported at the moment. We are examining workarounds like
```

Comment:
``particularly well supported`` -> new definition of support level ? Maybe this should say  simply ``not supported``